week-1/fix/median.js

```diff
@@ -2,10 +2,34 @@
 // Start by running the tests for this function
 // If you're in the week-1 directory, you can run npm test -- fix to run the tests in the fix directory
 
-function calculateMedian(list) {
+/* function calculateMedian(list) {
   const middleIndex = Math.floor(list.length / 2);
```

Comment:
Can you remove all commented code since your don't need it. Is not good practice to push commented code. You should always make sure to remove useless codes and comments, console.logs  before you push.